### ISSUE TITLE
Is rn-diff no longer a thing?

### ISSUE THREAD

**Issue by shercoder** [ncuillery/rn-diff#41]
Is https://github.com/pvinis/rn-diff-purge favorited over rn-diff? Is that why we are no longer seeing any new patches in rn-diff?

**Comment by ryankask** [comment]
I don't think anyone is maintaining this repository anymore. I wasn't able to use the script to generate the the new branches so I haven't been able to update the repo.

`rn-diff-purge` is active and I recommend you use that.

**Comment by pvinis** [comment]
After briefly talking to @ncuillery, he is a bit busy with things, so he didn't have time to maintain things here with git-upgrade.

I was using rn-diff but only while it was maintained, and when I tried to help I could only do it a few times, not consistently, because of troubles with git-upgrade and me not agreeing 100% with that way.

So, in the end, I stopped using that and started using the purge way. It has been working well for me, so I put it up. As others have said, it has no extra dependencies and the script will always run correctly, since it basically deletes the old dir, creates a new one, and then diffs. Feel free to try the purge repo and decide. :)
